str/trans-eqtl/trans_associatr_runner.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-function-docstring,no-member,too-many-arguments
+"""
+This script runs associaTR, given a chromosome or cell type.
+Ensure prior scripts have been run to generate dependent files, particularly:
+- get_cis_numpy_files.py
+- pseudobulk.py
+- get_covariates.py
+- qc_filters_associatr.py (depends on qc_annotator.py)
+
+ analysis-runner --dataset "bioheart" --config trans_associatr_runner.toml \
+    --description "run associatr" \
+    --access-level "test" \
+    --memory "8G" \
+    --output-dir "str/associatr/trans_pilot/bioheart_n975" \
+     python3 trans_associatr_runner.py
+
+
+"""
+import json
+
+import pandas as pd
+
+import hailtop.batch as hb
+
+from cpg_utils import to_path
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import get_batch, init_batch, output_path
+
+
+def main():
+    """
+    Run associaTR processing pipeline
+    """
+    b = get_batch(name='Run associatr')
+    init_batch()
```

Comment:
I don't think this is needed here? `init_batch` starts a hail query runtime, but you only need a batch